### ISSUE TITLE
update example button text and labels for better accessibility

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-confidenceSlider#81]
There are a couple of accessibility issues with the current `buttonText` and `ariaLabel` examples as well as inconsistency with other question components.

1. Use of '[clicking](https://github.com/adaptlearning/adapt-contrib-confidenceSlider/blob/bf38d28051989cca7f0e0746449190cbd550315a/example.json#L63)' isn't inclusive terminology as it assumes a mouse user.

```
            "_submit": {
                "buttonText": "Submit",
                "ariaLabel": "Submit your answer by clicking here."
            }
```

2. 'Show feedback' `buttonText` is unnecessary inconsistent with the `ariaLabel`. Voice activation users may have difficulty accessing this button with voice commands. Minor issue as 'feedback' is present in both but ideally the full visual text label 'show feedback' should be included in the `aria-label`.

```
            "_showFeedback": {
                "buttonText": "Show feedback",
                "ariaLabel": "View the question feedback again by clicking here."
            }
```

Aside from accessibility, it would make sense for button text and labels for the same functionality to be consistent. This is the case for other question components e.g. see [MCQ example.json](https://github.com/adaptlearning/adapt-contrib-mcq/blob/f49fe9a79ca9450ceb9a1fea14ac9e94205bc41b/example.json#L59), [Slider example.json](https://github.com/adaptlearning/adapt-contrib-slider/blob/608546a74e8892f8c140240ab6eb9756ac5231cb/example.json#L42).

**Suggestion:** update example `_buttons` as per other question component buttons.

e.g. 

```
        "_buttons": {
            "_submit": {
                "buttonText": "Submit",
                "ariaLabel": "Select here to submit your answer."
            },
            "_reset": {
                "buttonText": "Reset",
                "ariaLabel": ""
            },
            "_showFeedback": {
                "buttonText": "Show feedback",
                "ariaLabel": ""
            }
        }
```
